java/javax/inject/BUILD

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2017 The Dagger Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description:
+#   A JSR-330 compliant dependency injection system for android and java
+
+package(default_visibility = ["//:src"])
+
+filegroup(
+    name = "gwt",
+    srcs = glob(["**/*.gwt.xml"]),
+)
```

Comment:
Here's another idea so we don't need to have this extra BUILD file+indirection:

gwt/dagger/Dagger.gwt.xml
gwt/javax/inject/Dagger.gwt.xml
```python
genrule(
    name = "gwt",
    srcs = glob(["**/*.gwt.xml"]),
    outs = ["gwt.jar"],
    cmd = "jar cf $@ -C gwt . ",
)
```

Or they can live in java/dagger/gwt/ etc